BuildResidentialHPXML/measure.rb

```diff
@@ -3467,10 +3483,64 @@ def self.set_roofs(hpxml, runner, model, args)
 
   def self.set_rim_joists(hpxml, runner, model, args)
     model.getSurfaces.sort.each do |surface|
-      # TODO
+      next if surface.surfaceType != 'Wall'
+      next unless ['Outdoors'].include? surface.outsideBoundaryCondition
+
+      exterior_adjacent_to = HPXML::LocationOutside
+      interior_adjacent_to = get_adjacent_to(surface)
+
+      next unless (args[:geometry_rim_joist_height] - Geometry.get_surface_height(surface)).abs < 0.00001 # this is not a "wall"
+
+      if args[:rim_joist_assembly_r].is_initialized && (args[:rim_joist_assembly_r].get > 0)
+        insulation_assembly_r_value = args[:rim_joist_assembly_r]
+      end
+
+      if exterior_adjacent_to == HPXML::LocationOutside && args[:wall_siding_type].is_initialized
+        siding = args[:wall_siding_type].get
+      end
+
+      if args[:wall_color] == Constants.Auto && args[:wall_solar_absorptance] == Constants.Auto
+        solar_absorptance = 0.7
+      end
```

Comment:
This is weird, why are we hard-coding a value here?

Reply:
🤷‍♂️